tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/steps/PitclipseSteps.java

```diff
@@ -60,24 +59,24 @@
 public class PitclipseSteps {
 
     @When("test {word} in package {word} is run for project {word}")
-    public void runTest(final String testClassName, final String packageName, final String projectName) throws CoreException {
+    public void runTest(final String testClassName, final String packageName, final String projectName)
+            throws CoreException {
         // No need to do a full build: we should be now synchronized with building
         // Build the whole workspace to prevent random compilation failures
-        // ResourcesPlugin.getWorkspace().build(IncrementalProjectBuilder.FULL_BUILD, new NullProgressMonitor());
-        System.out.println
-            (String.format("Run PIT on: %s %s.%s",
-                projectName, packageName, testClassName));
+        // ResourcesPlugin.getWorkspace().build(IncrementalProjectBuilder.FULL_BUILD,
+        // new NullProgressMonitor());
+        System.out.println(String.format("Run PIT on: %s %s.%s", projectName, packageName, testClassName));
```

Comment:
Avoid reformatting existing code when the PR is not about reformatting: it can be confusing and adds unnecessary review work.